packages/core/addon/utils/chart.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * given a transform, returns the translation that transform provides
+ * @param {String} transform - transform to calculate
+ * @returns {Object} - Object with x and y properties
+ */
+
+export function getTranslation(transform) {
```

Comment:
Do we have tests for this?

Reply:
Is this method something we could pull in from `d3-interpolate` or is this somewhat different?